GCRCatalogs/alphaq.py

```diff
@@ -134,7 +135,7 @@ def _collect_native_quantities(name, obj):
                 'shear2',
             ),
             'magnification': (lambda mag: np.where(mag < 0.2, 1.0, mag), 'magnification'),
-            'halo_id':       'hostIndex',
+            'halo_id':       'hostHaloTag',
```

Comment:
Is this change backward compatible? If not, need to change the modifier for older catalog files.

Reply:
It is backwards compatible. But the older versions should remain the same. I've updated the GCR to use the older hostIndex for the older versions.